wolfHSM/Doxyfile

```diff
@@ -865,7 +865,7 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = README.md wolfhsm
+INPUT                  = README.md wolfhsm/wh_client.h wolfhsm/wh_server.h
```

Comment:
In `wolfHSM/Makefile` there is a typo.

```
- $(Q)cat src/appendix01.md build/pdf/wh__*8h*.md > build/html/appendix01.md
+ $(Q)cat src/appendix01.md build/html/wh__*8h*.md > build/html/appendix01.md
```

Did that not work for you?

Reply:
It works fine for me, in fact, weirdly it breaks the website when I apply that patch. The right sidebar with the function links disappears. Does it break for you as-is (with build/pdf)?

Reply:
I'm sure it would. The `perl -i` calls on the PDF are fancier, so it ends up being fine. Let's keep it as-is for now.